aiida/backends/djsite/db/migrations/0021_dbgroup_name2label_type2type_string.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
```

Comment:
I prefer that we call this `0021_dbgroup_name_to_label_type_to_type_string.py`

Reply:
changed